cedar-example-use-cases/discord/policies.cedar

```diff
@@ -0,0 +1,21 @@
+// Allow SendMessage when the user has the SendMessage permission
+permit (
```

Comment:
You can inline the `in` check into the scope:

```
permit (
  principal in Permission::"SendMessage",
  action == Action::"SendMessage",
  resource == Channel::"general"
);
```

We prefer to put these constraints in scopes because they can then be used for more efficient policy slicing.